### DISCUSSION TITLE
Questions Related to Mesh Parameters

### DISCUSSION THREAD

**Discussion by usmanshahid90** [mdolab/dafoam#146] [General]
I am working on mesh refinement and I have certain queries regarding that.  I am starting a discussion on it and will keep populating this discussion as I proceed.

1)  How growth ratio of mesh is being controlled in DAFOAM? Is there any parameter for it?
2) What is the meaning of march distance for extruding?
3) I want to increase the mesh density in the region after trailing edge. Which parameter I must be changing?

Till now, I have these questions. I will keep on adding the questions under this discussion.

Kindly guide me,
Regards,

**Comment by friedenhe** [comment]
I believe your questions have been answered in the workshop. The video recording is available  from our [YouTube channel](https://www.youtube.com/channel/UCr9SKeQBuntzYf-3ubtfzmQ)

**Reply by usmanshahid90** [reply]
yes. they have been answered. thanks
